Blueprints/BlueprintDefinitions/vs2022/EmptyFunction-Image-FSharp/template/src/BlueprintBaseName.1/Readme.md

```diff
@@ -22,9 +22,9 @@ Alternatively the Docker file could be written to use [multi-stage](https://docs
 have the .NET project built inside the container. Below is an example of building .NET 5 project inside the image.
```

Comment:
.NET 5 => .NET 6
or .NET alone may be better.

Reply:
Fixed